src/components/TokenImg.tsx

```diff
@@ -7,14 +7,19 @@ function _loadFallbackTokenImage(event: React.SyntheticEvent<HTMLImageElement>):
   image.src = unknownTokenImg
 }
 
-const TokenImg = styled.img.attrs(() => ({ onError: _loadFallbackTokenImage }))`
+interface TokenImgProps {
+  opaque?: boolean
+}
+
+const TokenImg = styled.img.attrs(() => ({ onError: _loadFallbackTokenImage }))<TokenImgProps>`
   width: 2.8rem;
   height: 2.8rem;
   border-radius: 3.6rem;
   object-fit: contain;
   margin: 0 1rem 0 0;
   background-color: white;
   padding: 2px;
+  opacity: ${(props): number => (props.opaque ? 0.4 : 1)};
```

Comment:
I think the logic of the flaf is inverred:

 `opaque definition: not able to be seen through; not transparent.`

I would call it `faded` or `deprecated`